store/src/config.rs

```diff
@@ -31,6 +32,8 @@ pub struct StoreConfig {
     pub endpoint: Endpoint,
     /// SQLite database file
     pub sqlite: PathBuf,
+    /// Genesis file
+    pub genesis_filepath: String,
```

Comment:
Is there a reason not to use `PathBuf` for `genesis_filepath` as well? I'm fine going with strings too - but I'd rather keep things consistent.

Reply:
I preferred `String` over `PathBuf`, because `PathBuf` doesn't convert to a `&str` nicely. For example, we'd need to write:

```rs
config.genesis_filepath.as_path().to_str().expect("path contains non UTF-8 characters")
```
`Path::to_str()` returns `Option<&str>`, which fails when the OS path contains non-UTF8 characters (which apparently some OSes support). So ultimately, I was forced to introduce an `expect("")`, which I prefer to avoid when possible.

`PathBuf` is nice to manipulate paths, which we don't need to do here. On the other hand, using a `Path` would be nicer than `String`...

Actually, I will change it to `PathBuf`, as even though we introduce the unfortunate `expect()`, I prefer the more representative type `PathBuf` as opposed to a general `String`.

Reply:
Switched to using `PathBuf`